Api/Data/FilmSearchResultInterface.php

```diff
@@ -3,16 +3,17 @@
 namespace Wtc\Film\Api\Data;
 
 use Magento\Framework\Api\SearchResultsInterface;
+use Wtc\Film\Api\Data\FilmInterface;
 
 interface FilmSearchResultInterface extends SearchResultsInterface
 {
     /**
-     * @return \Wtc\Film\Api\Data\FilmInterface[]
+     * @return FilmInterface[]
```

Comment:
Unfortunately in an inteface you do need the fully qualified interface name, so this change will need reverting